src/main/java/org/eaa690/aerie/communication/EmailSMSSender.java

```diff
@@ -0,0 +1,18 @@
+package org.eaa690.aerie.communication;
+
+import org.eaa690.aerie.service.PropertyService;
+
+public class EmailSMSSender implements MessageSender {
```

Comment:
It looks like the EmailSMSSender is some sort of variation of the already existing EmailService and SMSService.  why not have those services implement the MessageSender interface, and not have this class?